ios/brave-ios/Sources/PlaylistUI/MediaScrubber.swift

```diff
@@ -67,13 +82,23 @@ struct MediaScrubber<Label: View>: View {
                   ),
                   alignment: .leading
                 )
-                .animation(.linear(duration: 0.1), value: currentTime)
+                .osAvailabilityModifiers { content in
+                  if #available(iOS 17.0, *) {
+                    content.transaction(value: currentTime) { tx in
```

Comment:
Creating a linear animation with value acts differently on iOS 17 ?

Reply:
The reason is that in iOS 17 we get the API that lets us apply a transaction based on a value changed where as in 16 we can only apply an animation based on value changed. What this lets us do is actually see if the transaction that caused the change contains an animation already which lets us add our implicit linear animation if one doesnt exist but also lets you control the animation yourself if you want